doc/flame/inputs/gesture_input.md

```diff
@@ -473,6 +473,32 @@ class MyGame extends FlameGame with HasHoverables {
 ```
 
 
+### DoubleTapCallbacks
+
+
+Flame also offers a mixin named `DoubleTapCallbacks` to receive a double-tap event from the
+component. To start receiving double tap events in a component, add the
+`DoubleTapCallbacks` to your `PositionComponent`.
+
+```dart
+class MyComponent extends PositionComponent with DoubleTapCallbacks {
+  @override
+  void onDoubleTap(DoubleTapEvent event) {
```

Comment:
I thought `DoubleTapCallbacks` should match with the `DoubleTapDetector`.
But indeed aligning with `TapCallbacks` makes more sense since its a new API.